tests/kernel/bigquery_tests.py

```diff
@@ -49,15 +49,43 @@ class TestCases(unittest.TestCase):
   @mock.patch('google.datalab.Context.default')
   def test_udf_cell(self, mock_default_context, mock_notebook_environment):
     env = {}
+    mock_default_context.return_value = TestCases._create_context()
+    mock_notebook_environment.return_value = env
+
+    with self.assertRaises(Exception):
```

Comment:
Can you be more specific with the expected Exception here (and below) to make sure you are not catching the wrong exception?